python/ecl/ecl/__init__.py

```diff
@@ -9,4 +9,12 @@
 # Both of these legacy forms are controlled by the cmake switch
 # INSTALL_ERT_LEGACY.
 
+import warnings
+
+warnings.simplefilter('always', DeprecationWarning)
+warnings.warn(
+    "Importing from ert is deprecated and will be removed in a future release, use: \'from ecl import\' instead",
+    DeprecationWarning
+)
+
 from ert.ecl import *
```

Comment:
Will this not hit both deprication warnings because of this line?

Reply:
does not seems like it does..... and anyways - better having two deprecationwarnings instead of changing the implementation.